packages/core/src/client/index.ts

```diff
@@ -37,7 +40,8 @@ export class Web3InboxClient {
   public constructor(
```

Comment:
Should this be a private constructor? Seems like we are ok making a breaking change here

Reply:
This should have been private, and it's not documented to be used. Only the `static` method for `init` is documented

Reply:
I'm okay with making breaking changes here, this constructor is undocumented